pkg/server/datastore/sqlstore/models.go

```diff
@@ -123,6 +123,33 @@ func (DNSName) TableName() string {
 	return "dns_names"
 }
 
+// FederatedTrustDomain holds federated trust domains.
+// It has the information needed to get updated bundles of the
+// federated trust domain from a SPIFFE bundle endpoint server.
+type FederatedTrustDomain struct {
```

Comment:
@evan2645 has mused before about a way to designate a federated trust domain as "federated with" by default (from an entry point of view. If we're confident that we want to implement such a feature, we might want a column for that...

Reply:
That sounds good! I'm adding a new bool field `federate_with_new_registration_entries` for this purpose.